update-minor-versions.mjs

```diff
@@ -1,35 +1,47 @@
 import { execSync } from "child_process";
 import fs from "fs";
 
-function readJson(filepath) {
-  return JSON.parse(fs.readFileSync(filepath, "utf-8"));
-}
-
 function bumpStudioPlugin() {
   console.log("... bumping studio-plugin");
   execSync("npm version minor -w=packages/studio-plugin");
-  const packageJson = readJson("./packages/studio-plugin/package.json");
-  const newVersion = packageJson?.version;
-  if (!newVersion) {
-    throw new Error("Could not parse studio-plugin version from package.json");
-  }
-  return newVersion;
+  return readVersion("./packages/studio-plugin/package.json");
 }
 
-function bumpStudio(newVersion) {
+function bumpStudio(pluginVersion, uiVersion) {
   console.log("... bumping studio");
   const packageJsonPath = "./packages/studio/package.json";
   const packageJson = readJson(packageJsonPath);
   // `npm i @yext/studio-plugin@${newVersion} --save-exact` does not update the package json,
   // likely because the new version does not exist yet.
-  packageJson.dependencies["@yext/studio-plugin"] = newVersion;
+  packageJson.dependencies["@yext/studio-plugin"] = pluginVersion;
+  packageJson.dependencies["@yext/studio-ui"] = uiVersion;
   fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2));
   execSync("npm version minor -w=packages/studio");
 }
 
+function bumpStudioUI() {
+  console.log("... bumping studio-ui");
+  execSync("npm version minor -w=packages/studio-ui");
+  return readVersion("./packages/studio-ui/package.json");
+}
+
+function readJson(filepath) {
+  return JSON.parse(fs.readFileSync(filepath, "utf-8"));
+}
+
+function readVersion(packageJsonPath) {
+  const packageJson = readJson(packageJsonPath);
+  const newVersion = packageJson?.version;
+  if (!newVersion) {
+    throw new Error("Could not parse studio-plugin version from package.json");
```

Comment:
nit: can the error message be updated so it's not hard-coded for `studio-plugin`?

Reply:
updated!